File: Programmers/level1/harshadNumber/harshad_number.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/12947

def solution(x):
    answer = True
    sum_digits = 0
    for d in str(x):
        sum_digits += int(d)
    if x % sum_digits != 0:
        return False
    return answer

File: Programmers/level1/harshadNumber/harshadNumber.js
// https://programmers.co.kr/learn/courses/30/lessons/12947?language=javascript

export const solution = (x) => {
    let answer = true;
    let sum = 0;
    String(x).split('').forEach(char => sum += Number(char) );
    if(x % sum !== 0) return false;
    return answer;
}